docs/Concepts/Privacy/Privacy-Overview.md

```diff
@@ -9,6 +9,9 @@ Other participants cannot access the transaction content or list of participants
 !!! important
     For production systems requiring private transactions, we recommend using a network 
     with a consensus mechanism supporting transaction finality. For example, [IBFT 2.0](../../HowTo/Configure/Consensus-Protocols/IBFT.md).
+    
+!!! Caution
+    If you plan to use private transactions, do not use [pruning](../Pruning.md).
```

Comment:
Perhaps we can say that pruning isn't supported with private transactions? The current wording makes it sound like we don't enforce it and they're in charge of it.

Reply:
Updated.